src/Microsoft.Health.Dicom.Client/DicomWebClient.cs

```diff
@@ -29,7 +29,7 @@ public class DicomWebClient : IDicomWebClient
     {
         private const string TransferSyntaxHeaderName = "transfer-syntax";
 
-        private static readonly Uri BaseCustomTagUri = new Uri("/tags", UriKind.Relative);
+        private const string BaseExtendedQueryTagUri = "/tags";
```

Comment:
Let's keep this as a `Uri` unless there's a reason we need it as a `string`. There are `Uri` ctors for combining paths

Reply:
The original idea with Uri instead of string is that we can build other Uris by new Uri(Uri, relativePath), but I find it requires the Uri to be absolute while we always relative.

e.g: 
for delete tag 00100121, uri is "tags/00100121", i, new Uri(BaseExtendedQueryTagUri, "00100121") throws exception.